provisionerd/proto/version.go

```diff
@@ -14,7 +14,7 @@ import "github.com/coder/coder/v2/apiversion"
 //   - Add new field named `devcontainers` in the Agent.
 const (
 	CurrentMajor = 1
-	CurrentMinor = 4
+	CurrentMinor = 5
```

Comment:
Add comment to the set above

Reply:
wait what, I did already. where did it go??